routers/repo/download.go

```diff
@@ -27,6 +28,7 @@ func ServeData(ctx *context.Context, name string, reader io.Reader) error {
 
 	// Google Chrome dislike commas in filenames, so let's change it to a space
 	name = strings.Replace(name, ",", " ", -1)
+	name = path.Base(name)
```

Comment:
If possible for little optimization, maybe do `path.Base()` before `String.replace()`
It could even be done like `name = strings.Replace(path.Base(name), ",", " ", -1)`